docs/admin/admission-controllers.md

```diff
@@ -101,14 +112,16 @@ If your cluster supports containers that run with escalated privileges, and you
 restrict the ability of end-users to exec commands in those containers, we strongly encourage
 enabling this plug-in.
 
-### GenericAdmissionWebhook (alpha)
+### GenericAdmissionWebhook (alpha in 1.8; beta in 1.9)
```

Comment:
Why did this get called Generic, when we use the term Validating elsewhere.  Wasn't naming settled in the design phase?

Reply:
It is Validating. Will fix this when https://github.com/kubernetes/kubernetes/pull/55988 merges.